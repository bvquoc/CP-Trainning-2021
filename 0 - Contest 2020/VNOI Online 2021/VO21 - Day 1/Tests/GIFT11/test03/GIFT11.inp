10
200
1867 8565 3586 17 101425 9038 504333 1964 7625 42954676 60008339 459446 385213 93 89 31094950 31 553513 421355 5142 99015482 54 50 2741 511510 62851932 91053531 5382 30076241 8920 41083839 90692713 88291853 12690358 719947 825199 2842 155560 8169 31 7761 27 987170 361945 681676 93 21 194238 57738835 19845278 5680 21 33709888 4545 5856 98 31 707292 21136551 21653662 561805 184745 57 2246 6792 40602880 60 281737 6972 35621651 43985422 55165752 99 23 560522 62074469 21 6657 81824570 42024415 270787 69 99 33 47927647 62 99 844131 67 919429 71 92 54 43 644756 2883 71095340 14697395 11 48057771 41 31 38757459 40870624 7885 20948784 58461610 1193 6790 67925791 389342 433577 881163 86975236 22945422 56017696 44 92665425 7827 83035983 86644983 5726 7056 68390023 82166992 569542 4573 19201881 373601 89 42 3703 95313451 663909 74 26078416 14 716119 87273252 29748055 62758064 267510 388666 81967804 55110861 17792213 83517206 2984 92824935 79 33316599 26 21044127 39 232606 95215241 3951 61 64443214 247793 11 72568020 23 85 511624 710635 73560842 751850 49138052 92098534 5479 5374 41532804 57032503 40832560 963288 341351 723406 39 33 84139984 714621 32667595 92 224224 201941 421422 19 391607 20425190 7745 43 63103094 48799426 82433796 52539273 9479 3762 43218938 629590
200
83611954 512286 4531 304430 50 40 93297695 66281021 28277979 2705 99 40319051 91802323 79946785 9104 25832106 19926522 94 3734 922057 2048 850564 62 4531 38201622 63 27 61964432 7894 62 46324132 857497 422136 75527999 7624 4831 710453 913906 69 13242027 7447 77615804 343844 91896726 55 59347341 136894 2034 608780 97623063 8703 340102 9025 65477963 4145 3349 987311 685913 19 75221407 8502 5435 9991 97 68345239 93511178 2701 96 46261524 332654 541704 20 8925 86269617 8848 973994 127651 23586211 45 310035 54 11585238 77086834 3213 14 35146939 150766 53 61697633 48237314 125363 93245368 2231 157213 407458 7659 302735 5389 45415068 931283 298059 84 548546 46863586 33944331 70587662 75 528679 95 10492898 67652437 45302293 41604509 1942 16882693 40 980944 7852 1980 693549 642668 29421332 36229173 3105 7553 10 8511 72537970 89807968 28734681 49091052 361959 91 95548672 657493 69 17 820149 30323891 6845 67 80365257 716359 94484896 643450 3119 9961 5628 79 6470 72 18385243 613392 980111 3663 789476 5532 67 344132 51 21 1838 850037 9220 97 6822 60785697 31926077 329139 2524 19865226 99 7143 42 65 4445 421201 42 859478 70 71 987745 88 95 76165997 441811 392323 63572919 71624272 552754 5739 2866 12 75 3606 30 8591 696149 2006 7507
200
34890845 85260189 42333530 24651413 27712112 32 40726356 38887314 349025 27545707 644693 187200 157938 8220 41858462 772240 50881955 28 313647 1244 97128053 831616 75 988060 98812074 58619812 51946033 4545 293282 21855742 14940644 31452348 20523007 18261278 43476131 8334 59133345 9070 29907056 8016 28 83789879 10560908 74274483 40811641 903729 7792 51903833 2650 10485366 27 76 45 6143 792451 20 74383007 77431524 74860645 6672 600151 77555791 573403 157832 3003 756489 5917 2272 47900012 92 5797 436199 4691 633036 293532 10028560 84 59748791 63 52681450 443738 7349 67 99177992 72427402 8519 65762165 8386 89263470 7301 37314578 85946897 7708 244297 5482 61536862 93 42 50 746163 325711 74724278 3731 29 84 16309257 887302 7453 4884 786406 936440 55 888283 95 12922731 8616 95 8304 629088 24150231 47 94 886392 7947 75176586 29993069 2115 772701 68 80993049 52577236 2463 110412 790548 52 51909984 277882 58446480 68 67508195 767655 678142 460317 7704 503413 82 921723 20 4548 4610 92282883 4228 3997 4051 550166 28252142 65644682 738617 94554133 21668005 51 508350 823681 46959604 52240610 3251 21 61 4902 990944 51 85113819 53831443 863359 1852 778613 48 92 35 72652714 347494 63 92 912395 12543982 64010111 5910 41233258 6730 5351 8088 23309366 61171298 64385869 89 91766306 712752 569290 74 68185795
200
24 35 83893495 509659 72 63 1411 82911498 981524 6803 39 8364 75 76023698 76184054 879363 2112 56 835007 593966 833011 9365 48 98 22179867 78 85547462 14272857 48003928 85 34 67369050 95450539 55402522 328005 116036 418326 593980 330275 1985 60517081 5247 127683 19045144 16079669 53791807 76 403076 9548 77211523 53697801 9484 60177822 38 75 17 6132 77909413 2230 10196761 890552 2709 1009 815391 39981411 4677 325803 62564855 62436876 3039 82437549 18959568 75743958 123020 75 7461 2130 218709 51192524 53 99645712 59452706 19835788 2411 688833 87067909 61992023 589535 57 11799406 34219503 98260962 1586 4191 2565 65678957 37 25650777 24513098 53377938 3188 4580 636815 66432224 6597 34260458 61 3890 60316968 39034388 69 4064 38737485 236170 79425031 664595 307390 47636382 47543510 71279316 87420634 13287713 33 4892 486404 21 437170 5229 14428509 25061590 75 7654 97 51225770 5379 98 80607069 7601 20 764161 9375 94 46582406 413884 69028323 314843 64227708 10 97941139 2560 7614 351826 17 776348 275516 68228751 12 1591 41792585 33 4792 655611 442420 98387683 87 981761 94276927 7532 27577830 2153 1075 41587040 540863 377837 910809 24 20135763 263094 42 1006 15092161 64 82324351 5201 834156 67835969 16658808 53275507 16646581 386053 87949016 52089390 3556 74809738 10 341754 16 679425 14 7128
200
8976 82 28786596 51 55073488 1023 69 695822 48496155 17 87072337 622920 3145 873568 302761 41 969851 979086 95 7384 554226 20728596 28 87469184 2456 1242 2841 225913 397654 35138218 94588051 26107260 53 991324 82062229 807866 80 568106 36034057 11 63158460 7157 20304304 27980940 82632937 30 49 10 96812642 65105157 4155 26638678 584937 389515 6513 18717851 47369854 59135203 42 44742945 30 400316 60 269946 356055 57 3928 807925 95635518 4399 6271 479684 85477942 66302074 43141257 75407193 8138 7780 83 6676 618947 6716 90534850 87290838 8992 1436 1731 3800 974725 3380 4787 32641011 865416 44086263 325771 5008 6964 23 66322019 78 68 38 997534 8349 2449 829977 27 1207 37800821 2555 6716 84714255 1124 573517 437666 74 2297 40400246 94999246 302829 76 70294623 91064001 75415629 83830983 938714 43812092 5287 91837130 50292056 225376 92761989 15 497738 89 61253247 68 1296 9134 20 1416 1412 68893066 1571 790607 8390 465915 19 98573955 718280 465499 69 84161661 48477755 77 143024 69091391 396999 10 66 2417 52 771029 37238491 2493 87512017 62325334 13 981403 56 275562 20898165 7308 791469 525265 75 77081722 984377 79 62575497 4781 35668228 180448 40498914 109135 88012288 9745 581309 12 2941 52319626 53126345 83 16175181 81999076 9897 34 571073 669784 69059716
200
85 71 5646 9006 7730 6255 6761 37 87 95394268 201848 77760428 34856785 252808 926110 37896426 421690 95669293 59476674 80 127546 567836 96 22 40 4357 77 61880609 859364 16 22 1666 24 1348 6204 7088 35616184 1419 96 15758996 552124 69 499516 77353432 17 76 82913237 96882236 33499805 367844 33756724 89035920 62774973 3468 7500 69 56 53071846 13067372 52537745 932125 139772 1142 525119 2793 42746007 1051 99 19808583 75372871 53406588 22065388 51167410 10 83269274 16507917 4763 38133709 4261 98 40563418 46 3120 52253208 7488 5589 53811914 82 7471 68069968 8509 13 5503 41 183027 89768478 42089290 56621067 23279027 191599 911216 8781 166087 699085 764130 35 8184 99668043 6946 406380 19897902 28315321 86 389517 15232644 4136 23382029 6023 10 236649 70165563 30529699 867512 13080589 30 22666140 155000 34496358 42 6528 5891 31 705297 5339 605931 862295 101002 74 5599 92 47 5493 12 67543397 826814 94 96848737 27 147003 246770 533522 67277368 5880 814143 23 71075481 56 39 764567 2383 32717351 3007 65 45 21855883 44 719720 74 53767692 62751221 79 55295842 913738 35 1316 72063322 46 224953 9008 89021924 4818 83 7819 314127 1518 87309700 87855144 67860142 14 521247 95255311 16982352 15629083 442270 809632 94 963263 9843 69 684139
200
91714054 50 86438728 732187 3387 81 40105056 7904 3239 9517 6207 62684303 10133122 33001106 86186942 3095 3469 883762 1150 19678411 14113746 824430 4311 89 434249 58 1740 1360 745547 66879100 63831820 5755 277178 99 197954 962525 189686 7549 281517 8272 31458625 48 91457743 8283 53 38 61 15546137 7653 4601 76910940 1130 11 42 47 55635551 53854735 67 69 400529 77595184 33 7276 80388345 1766 72 77 66 84 388018 519507 358384 95 1263 1072 33467009 152076 36189495 2134 40891979 20 331785 94 28007329 2121 5743 3196 88514451 55880334 98711394 50 2883 275343 83701194 95 104208 3820 83 13140481 24627836 39494982 11415783 28 1473 59 6430 8499 872263 470167 1015 774827 7719 72184368 5458 2764 41487834 214506 79 53871399 17803153 6838 199253 5357 961661 74133318 542126 79 87460336 8770 78 88 317657 64698420 86377542 96118634 89 66 86 23027966 10 2523 705376 39 97 47 7556 32 780763 4940 529572 966044 501976 96827160 1377 6741 2153 22 198460 64 10564185 10102050 806101 4276 54462720 81942714 570317 238044 40 81 5991 78776746 19718382 87823416 8186 93826577 63 35 3095 5296 1998 865143 29 19792842 4271 30492706 175190 72375006 52012014 320576 86 81567278 104171 3970 850692 93396219 76754063 3538 348222 90 69
200
746168 66 28 95756624 124526 8269 82952985 9967 6708 47 60 405113 561795 68 90 70883008 696389 47 37284160 24 514818 32095953 4579 57289222 431676 83 4571 787847 2147 46 22680649 8480 7856 46452691 33599470 23736506 13234045 70297504 624603 14064654 78729714 2776 1616 164470 76012712 36 96 2786 73 822364 5394 31 4893 32 837841 8602 70796766 745598 36 70524616 981252 32753242 34212869 382670 903505 25 24166533 67 63402373 837597 6181 51 217377 866406 91530806 84564092 2894 67193995 69466921 94 11765078 831452 585102 12914490 333746 25 20117384 17 8386 16897513 96 52 63 25094918 46556154 15 615186 700788 233877 15131178 6517 73 34 65397416 579337 544613 44151316 40 525373 9379 367930 1192 7565 43 90306235 17 400447 576774 66 40 42 8572 62585617 5749 52 62574811 5288 27082805 69992365 4129 41 56319919 41 59749682 593781 2771 90 11 6092 57372386 879342 73634868 17 429981 92412556 10 9417 59 79 53 44 653419 35492668 1241 6693 5793 82780655 20 655824 46711319 97 34224888 76255393 9349 22741938 406570 741722 88356454 62 58894270 92719292 97561954 73 54 89664648 693027 1828 17 71 9728 5365 75 591586 227027 981602 29929086 72 7950 54657701 71642111 96 81 3470 98 76 8373 76 839371 69362169 6871
200
11637335 98 232966 568492 16000761 1017 61816133 839473 505701 8695 5983 517940 55662781 3059 79 878500 51265889 55 1664 7828 395251 90104014 4313 19633588 214419 79765889 73970961 19 8211 64895157 62 61562056 9451 69676443 761544 5326 40919508 278054 82740667 1465 58950903 30 398152 84821016 1311 32956713 2073 64636510 974650 229892 74506551 87958603 91 913191 55 983376 24 86 28 6534 27864385 7384 16829792 33 1800 57 18802948 57321882 7560 33 53 919178 93 49584347 601618 507366 25935481 7504 17945662 33 67053291 7669 2631 539563 594875 58626409 6942 2341 962058 70070317 1060 7433 70993399 59 58545898 8438 621702 949084 8993 98907711 38770901 85462377 57 290131 109959 78890394 486133 764701 93648892 21 84 46613805 68 22745518 44477472 306324 59810037 29 14571370 87 416730 55 667171 43201036 35 66 582156 30 4141 20 208134 34 35 9864 791602 52715878 26413316 99300930 242576 7981 5233 826426 8555 7946 72864217 47 18 29411334 7488 473799 51 820315 8420 8070 328764 12133584 16 548022 20 9872 6980 3393 368045 17 3194 8392 89368651 5154 60188090 359187 81414693 36263430 19988558 821789 51 6992 3254 2340 31637485 99 8043 706532 5014 45772219 560807 58 902829 42153502 368114 53 20768860 4354 966421 5353 24 65073634 878994 11000769 10 59826990
200
70 5254 9475 75849304 31397404 94553160 30851628 780583 73139883 45982561 22172041 171237 39664728 5513 2789 342788 7307 527698 24052279 416916 25005517 27 30 62737990 275159 74 89 15 43 4451 488298 80 31387058 351313 84301242 70038962 557892 456764 66564214 4215 278138 545326 62821918 464210 8437 121016 83 99676886 757591 35755082 36386682 357299 245441 4342 90 345498 860818 9340 25476566 13132412 7382 2230 2038 5227 33425139 51 45869370 57617046 6188 717786 171320 74999216 95403489 309765 191788 75 40 70465935 75171542 6705 6389 60930315 352823 745691 75535693 7880 777295 155160 633382 41 77978081 67 71530033 41 82 610890 6053 190323 23492658 42891907 590525 32 32 2074 46726402 63973316 847859 1376 41213084 93902928 84 74 78 8013 669068 6837 77034163 91 16078056 38701726 27442995 650923 70005860 93 338157 773536 229603 992467 537664 56776529 92339748 13 8120 89 89015977 6781 4443 91980429 18409523 260592 3236 46 80232771 48661091 91 6895 13799450 59795623 446962 4830 67529145 554894 3737 8208 67786337 23 7306 831806 53551122 35931973 2717 453464 29 7712 9931 82187496 33 19 1854 94588606 9188 79 41825990 638060 53779856 89 8940 528643 73770306 92 65270602 53 66633028 7750 926744 92 77356118 61807261 5152 862149 3675 12072767 7570 5592 88794867 412799 643679 91 1286 28824566
