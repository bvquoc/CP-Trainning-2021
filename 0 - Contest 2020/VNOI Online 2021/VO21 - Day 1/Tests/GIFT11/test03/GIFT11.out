245648024
245648024
245648024
245648024
245648024
245648024
245648024
245648024
245648024
245648024
